contrib/docker-compose/standalone/hockeypuck/etc/hockeypuck.conf.tmpl

```diff
@@ -4,7 +4,10 @@ indexTemplate="/hockeypuck/lib/templates/index.html.tmpl"
 vindexTemplate="/hockeypuck/lib/templates/index.html.tmpl"
 statsTemplate="/hockeypuck/lib/templates/stats.html.tmpl"
 webroot="/hockeypuck/lib/www"
-version="2.1.0"
+hostname="FQDN"
```

Comment:
Templating this template (maybe with envsubst?) might further simplify site customization & deployment steps.